scripts/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+ZIP_NAME="deployment.zip"
+
+pip install --target ./reqs -r sleuth/requirements.txt
```

Comment:
Might wanna check the version of python/pip are installed that we want. Additionally, we might want to eventually turn this project to using pyenv or poetry to better manage python requirements/builds.